### PR TITLE
Add Shivendu-Kumar.json

### PR DESCRIPTION
**Pull request by Shivendu-Kumar**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1309"></a>

### CODE REVIEW

File: public/data/Shivendu-Kumar.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "url": "https://twitter.com/shivendu_kr_jha",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/shivendu-kumar-731a46214/",
+        "icon": "linkedin"
+      }
+    ]
+
```
Comment (kumarsonsoff3) [inline public/data/Shivendu-Kumar.json:23]: Rest looks good to me! 

```suggestion
```

File: public/data/Shivendu-Kumar.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/shivendu-kumar-731a46214/",
+        "icon": "linkedin"
+      }
+    ]
+ 
+  }
```
Comment (kumarsonsoff3) [inline public/data/Shivendu-Kumar.json:24]:
```suggestion
}
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]

File: public/data/Shivendu-Kumar.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        "url": "https://twitter.com/shivendu_kr_jha",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/shivendu-kumar-731a46214/",
+        "icon": "linkedin"
+      }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/Shivendu-Kumar.json:23]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Shivendu-Kumar** [comment]
Please let me know if there is any problem with merging my PR.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by Shivendu-Kumar** [comment]
> LGTM 🚀 Thank you for your contribution! 😄👍
Thank You for the approval 😊

**Pull request merged**